Lower deps seems to fail:

> FAILED tests/test_integrations/test_integration_with_sentencetransformers.py::test_sentence_transformer_integration_cross_encoder[cross-encoder/ms-marco-TinyBERT-L2-v2-task1] - AttributeError: 'CrossEncoder' object has no attribute 'model_card_data'

--- a/mteb/models/model_meta.py
+++ b/mteb/models/model_meta.py
@@ -653,7 +653,9 @@ class ModelMeta(BaseModel):  # noqa: PLR0904
                 else None,
                 framework=["Sentence Transformers", "PyTorch"],
                 n_embedding_parameters=n_embedding_parameters,
-                adapted_from=_get_source_model(model.model_card_data),
+                adapted_from=_get_source_model(model.model_card_data)
+                if hasattr(model, "model_card_data")
+                else None,
             )
         )
 
@@ -707,7 +709,9 @@ class ModelMeta(BaseModel):  # noqa: PLR0904
                 n_embedding_parameters=cls._get_n_embedding_parameters_from_sentence_transformers(
                     model
                 ),
-                adapted_from=_get_source_model(model.model_card_data),
+                adapted_from=_get_source_model(model.model_card_data)
+                if hasattr(model, "model_card_data")
+                else None,
             )
         )
 

